Homarus/src/Controller/HomarusController.php

```diff
@@ -117,9 +114,12 @@ public function convert(Request $request)
         if ($format == "mp4") {
```

Comment:
At this point I'm unsure if there's a particular reason that `mp4`s get their own separate flow of logic with arguments where as [Houdini](https://github.com/Islandora/Crayfish/blob/2.x/Houdini/src/Controller/HoudiniController.php#L115) allows for the header to pass through everything that's to be ran.